Analyses/src/BFieldPlotter_module.cc

```diff
@@ -79,15 +80,15 @@ namespace mu2e {
     if(_axisOneMin >= _axisOneMax || _axisTwoMin >= _axisTwoMax) {
       throw cet::exception("BADCONFIG") << "BField mapping plane ill defined: "
 					<< _axisOneMin << " < axisOneValues < " << _axisOneMax << ", "
-					<< _axisTwoMin << " < axisTwoValues < " << _axisTwoMax;
+					<< _axisTwoMin << " < axisTwoValues < " << _axisTwoMax << endl;
     }
```

Comment:
The exception system strips std::endl.  If you want to force a break use "\n".

Reply:
Good to know, I'll simply let it not end the line as it originally was in that case.